### ISSUE TITLE
Fails to execute script, because its MIME type ('text/plain') is not executable.

### ISSUE THREAD

**Issue by siliconsorcery** [dart-lang/angular#1412]
Following the [Get Started](https://webdev-dartlang-org-dev.firebaseapp.com/guides/get-started) guide for Angular 5.0.0-alpha+14 and Dart 2.

* [ ] Dart SDK Version (`dart --version`)
Dart VM version: 2.0.0-dev.62.0 (Wed Jun 13 16:50:22 2018 +0200) on "macos_x64"

* [ ] AngularDart Version (i.e. `4.0.0`, `5.0.0-alpha+5`, `MASTER`)
Angular 5.0.0-alpha+14

* [ ] Whether you are using Windows, macOS, or Linux (if applicable)
macOS

* [ ] Whether you are using Chrome, Safari, Firefox, Edge (if applicable)
Chrome

* [ ] Any other packages or constraints we should know about
Using the CLI tools, webdev & stagehand.

## pubspec.yaml
```
name: angular_app
description: A web app that uses AngularDart 
version: 0.0.1

environment:
  sdk: '>=2.0.0-dev.56.0 <=2.0.0-dev.62.0'

dependencies:
  angular: ^5.0.0-alpha+14

dev_dependencies:
  angular_test: ^2.0.0-alpha
  build_runner: ^0.8.9
  build_test: ^0.10.2
  build_web_compilers: ^0.4.0
  test: ^0.12.30
```

## pub get
```
Resolving dependencies...
Overriding the upper bound Dart SDK constraint to <=2.0.0-dev.62.0 for the 
following packages:
...
```
webdev serve
## Errors from Chrome Inspector
```
main.dart.js:1 Failed to load resource: the server responded with a 
status of 404 (Not Found)

localhost/:1 Refused to execute script from 'http://localhost:8080/main.dart.js' 
because its MIME type ('text/plain') is not executable, and strict MIME type 
checking is enabled.
```
## Result 
The 'todo' example never stops displaying 'Loading...'

**Comment by leonsenft** [comment]
@siliconsorcery Can you check your console for errors after running `webdev serve`? The MIME type error is generally a sign that your application didn't compile successfully.

**Comment by siliconsorcery** [comment]
$ brew upgrade dart
```
Error: dart-lang/dart/dart 2.0.0-dev.63.0 already installed
```

$ dart --version
```
Dart VM version: 2.0.0-dev.63.0 (Fri Jun 15 00:42:43 2018 +0200) on "macos_x64"
```

$ pub global activate stagehand
```
Package stagehand is currently active at version 3.0.0.
Resolving dependencies... (2.5s)
+ args 1.4.3
+ async 2.0.7
+ charcode 1.1.1
+ collection 1.14.10
+ http 0.11.3+16
+ http_parser 3.1.2
+ path 1.6.1
+ source_span 1.4.0
+ stagehand 3.0.0
+ string_scanner 1.0.2
+ typed_data 1.1.5
+ usage 3.4.0
Precompiling executables... 
Precompiled stagehand:stagehand.
Installed executable stagehand.
Activated stagehand 3.0.0.
```

$ mkdir second
$ cd second
second$ stagehand web-angular
```
Creating web-angular application `second`:
  /Users/john/My/Projects/Learning/AngularDart/second/.gitignore
  /Users/john/My/Projects/Learning/AngularDart/second/CHANGELOG.md
  /Users/john/My/Projects/Learning/AngularDart/second/README.md
  /Users/john/My/Projects/Learning/AngularDart/second/analysis_options.yaml
  /Users/john/My/Projects/Learning/AngularDart/second/lib/app_component.css
  /Users/john/My/Projects/Learning/AngularDart/second/lib/app_component.dart
  /Users/john/My/Projects/Learning/AngularDart/second/lib/app_component.html
  /Users/john/My/Projects/Learning/AngularDart/second/lib/src/todo_list/todo_list_component.css
  /Users/john/My/Projects/Learning/AngularDart/second/lib/src/todo_list/todo_list_component.dart
  /Users/john/My/Projects/Learning/AngularDart/second/lib/src/todo_list/todo_list_component.html
  /Users/john/My/Projects/Learning/AngularDart/second/lib/src/todo_list/todo_list_service.dart
  /Users/john/My/Projects/Learning/AngularDart/second/pubspec.yaml
  /Users/john/My/Projects/Learning/AngularDart/second/test/app_test.dart
  /Users/john/My/Projects/Learning/AngularDart/second/web/favicon.png
  /Users/john/My/Projects/Learning/AngularDart/second/web/index.html
  /Users/john/My/Projects/Learning/AngularDart/second/web/main.dart
  /Users/john/My/Projects/Learning/AngularDart/second/web/styles.css
17 files written.

--> to provision required packages, run 'pub get'
```

second$ pub get
```
Resolving dependencies... (18.1s)
Overriding the upper bound Dart SDK constraint to <=2.0.0-dev.63.0 for the following packages:

args, boolean_selector, build_config, charcode, cli_repl, cli_util, convert, fixnum, glob, graphs, http, http_parser, intl, io, js, logging, meta, multi_server_socket, node_preamble, package_config, perf_api, plugin, pub_semver, quiver, quiver_hashcode, sass, shelf_packages_handler, source_map_stack_trace, source_maps, source_span, stack_trace, string_scanner, term_glyph,tuple, typed_data

To disable this you can set the PUB_ALLOW_PRERELEASE_SDK system environment variable to `false`, or you can silence this message by setting it to `quiet`.
+ analyzer 0.32.1
+ angular 5.0.0-alpha+14
+ angular_ast 0.5.3+2
+ angular_compiler 0.4.0-alpha+14
+ angular_components 0.9.0-alpha+14
+ angular_forms 2.0.0-alpha+6
+ angular_test 2.0.0-alpha+12
+ archive 2.0.0
+ args 1.4.3
+ async 2.0.7
+ bazel_worker 0.1.10
+ boolean_selector 1.0.3
+ build 0.12.6
+ build_config 0.2.6+2 (0.3.0 available)
+ build_modules 0.2.2+6
+ build_resolvers 0.2.0+2
+ build_runner 0.8.9 (0.8.10 available)
+ build_test 0.10.2+4
+ build_web_compilers 0.4.0+4
+ built_collection 3.1.1
+ built_value 5.5.1
+ charcode 1.1.1
+ cli_repl 0.1.3
+ cli_util 0.1.3
+ code_builder 3.1.0
+ collection 1.14.10
+ convert 2.0.1
+ crypto 2.0.3
+ csslib 0.14.4
+ dart2_constant 1.0.1+dart2
+ dart_internal 0.1.1
+ dart_style 1.0.14
+ fixnum 0.10.7
+ front_end 0.1.1
+ glob 1.1.5
+ graphs 0.1.2
+ html 0.13.3+1
+ http 0.11.3+16
+ http_multi_server 2.0.5
+ http_parser 3.1.2
+ intl 0.15.6
+ io 0.3.2+1
+ js 0.6.1
+ json_annotation 0.2.8
+ json_rpc_2 2.0.8
+ kernel 0.3.1
+ logging 0.11.3+1
+ matcher 0.12.2+1 (0.12.3 available)
+ meta 1.1.5
+ mime 0.9.6+1
+ multi_server_socket 1.0.1
+ node_preamble 1.4.2
+ observable 0.22.1+3
+ package_config 1.0.3
+ package_resolver 1.0.3
+ path 1.6.1
+ perf_api 0.1.0
+ plugin 0.2.0+2
+ pool 1.3.5
+ protobuf 0.8.0 (0.9.0 available)
+ pub_semver 1.4.1
+ quiver 0.29.0+1
+ quiver_hashcode 1.0.0
+ sass 1.6.0
+ sass_builder 2.0.2
+ scratch_space 0.0.2+1
+ shelf 0.7.3+1
+ shelf_packages_handler 1.0.3
+ shelf_static 0.2.7+1
+ shelf_web_socket 0.2.2+2
+ source_gen 0.8.2
+ source_map_stack_trace 1.1.4
+ source_maps 0.10.5
+ source_span 1.4.0
+ stack_trace 1.9.2
+ stream_channel 1.6.7+1
+ stream_transform 0.0.14
+ string_scanner 1.0.2
+ term_glyph 1.0.0
+ test 0.12.41
+ tuple 1.0.1
+ typed_data 1.1.5
+ utf 0.9.0+4
+ vm_service_client 0.2.4+3
+ watcher 0.9.7+8
+ web_socket_channel 1.0.8
+ yaml 2.1.14
Changed 87 dependencies!
Precompiling executables... (3.1s)
Precompiled build_runner:graph_inspector.
Precompiled test:test.
Precompiled build_runner:build_runner.
```
second$ pub global activate webdev
```
Package webdev is currently active at version 0.2.2.
Resolving dependencies... (3.7s)
+ args 1.4.3
+ charcode 1.1.1
+ collection 1.14.10
+ io 0.3.2+1
+ meta 1.1.5
+ path 1.6.1
+ pub_semver 1.4.1
+ source_span 1.4.0
+ stack_trace 1.9.2
+ string_scanner 1.0.2
+ webdev 0.2.2
+ yaml 2.1.14
Precompiling executables...
Precompiled webdev:webdev.
Installed executable webdev.
Activated webdev 0.2.2.
```

second$ webdev serve
```
[INFO] Setting up file watchers completed, took 61ms
[INFO] Waiting for all file watchers to be ready completed, took 245ms
[INFO] Reading cached asset graph completed, took 1.0s
[INFO] Checking for updates since last build completed, took 1.5s
[INFO] Running build completed, took 763ms
[INFO] Caching finalized dependency graph completed, took 486ms
[INFO] Succeeded after 1.3s with 0 outputs (0 actions)
Serving `web` on http://localhost:8080
Serving `test` on http://localhost:8081
```
## Errors from Chrome Inspector
```
main.dart.js:1 Failed to load resource: the server responded with a 
status of 404 (Not Found)

localhost/:1 Refused to execute script from 'http://localhost:8080/main.dart.js' 
because its MIME type ('text/plain') is not executable, and strict MIME type 
checking is enabled.
```

## Result
The 'todo' example never stops displaying 'Loading...'

**Comment by chalin** [comment]
Try using SDK 2.0.0-dev.61.0 (cf. #1406).

**Comment by matanlurey** [comment]
As mentioned, this is a result of failed build, and not related to script execution.

**Comment by siliconsorcery** [comment]
Thanks,

With some effort I installed 'SDK 2.0.0-dev.61.0' and was able to get the 'todo' application running.

Archive versions of the SDK are available from. [https://www.dartlang.org/tools/sdk/archive](https://www.dartlang.org/tools/sdk/archive)

**Comment by leonsenft** [comment]
Note we did just release +15 which should be compatible with dev.62+, so you shouldn't need to use the archived versions any longer after upgrading.

**Comment by siliconsorcery** [comment]
Thanks, everything is working now.

I used brew to uninstall and untap dart. Removed my path overrides for the old SDK.
> brew update
> brew cleanup
> brew doctor
> brew uninstall dart
> brew untap dart-lang/dart

Used brew to tap and install dart development SDK 2.0.0-dev.63.0
> brew tap dart-lang/dart
> brew install dart --devel

Bumped pubspec.yaml to use release +15 for the todo app
```
dependencies:
 angular: ^5.0.0-alpha+15
 angular_components: ^0.9.0-alpha+15
```
Grabbed the dependencies
> pub get
And finally ran,
> webdev serve
 and after 3m 46s of building Chrome connected to localhost:8080 and displayed the working 'todo' app.
